bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -5937,6 +6047,52 @@ private static void closeStatement(Statement statement) {
 		}
 	}
 
+	void setIngestJobEndDate(int ingestJobId, long endDate) throws TskCoreException, TskDataException {
+		CaseDbConnection connection = connections.getConnection();
+		acquireSharedLock();
+		ResultSet resultSet = null;
+		try {
+			Statement statement = connection.createStatement();
```

Comment:
Let's not make this a set once method. We can eliminate the TskDataException throw and the SELECT. This will keep our options open for job recovery.